packages/api/src/brevo-contact/contact-importer-error.interceptor.ts

```diff
@@ -0,0 +1,21 @@
+import { BadRequestException, CallHandler, ExecutionContext, Injectable, NestInterceptor } from "@nestjs/common";
+import { catchError, Observable } from "rxjs";
+
+import { ImporterError } from "../common/importer.error";
+
+@Injectable()
+export class ImporterErrorInterceptor implements NestInterceptor {
```

Comment:
Do we need an extra interceptor and error? Why not throw a CometValidationException?